makefile.shade

```diff
@@ -16,9 +16,16 @@ k-standard-goals
 #benchmark-tests .compile
   @{
     var sql = string.IsNullOrEmpty(E("BENCHMARK_SQL")) ? string.Empty : "-q " + E("BENCHMARK_SQL");
-    var scenarios = new string[] { "plaintext", "json", "mvcplaintext", "mvcjson" };
+    var scenarios = new string[]
+    {
+      "-n plaintext",
+      "-n plaintext --connectionFilter Benchmarks.PassthroughConnectionFilter",
```

Comment:
Another thought, should we add a "display name" or something like that? I'm not sure we want to surface plaintext & plaintext + connection filter as the same thing. I'm happy to treat this as a future enhancement